src/sentry/integrations/discord/webhooks/base.py

```diff
@@ -1,33 +1,16 @@
 from __future__ import annotations
 
 from django.views.decorators.csrf import csrf_exempt
-from rest_framework import status
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import Endpoint, region_silo_endpoint
-from sentry.integrations.discord.message_builder.base import (
-    DiscordMessageBuilder,
-    DiscordMessageFlags,
-)
 from sentry.integrations.discord.requests.base import DiscordRequest, DiscordRequestError
-from sentry.integrations.discord.views.link_identity import build_linking_url
-from sentry.integrations.discord.views.unlink_identity import build_unlinking_url
+from sentry.integrations.discord.webhooks.command import DiscordCommandHandler
+from sentry.integrations.discord.webhooks.message_component import DiscordMessageComponentHandler
 from sentry.web.decorators import transaction_start
 
-from ..utils import logger
-
-LINK_USER_MESSAGE = "[Click here]({url}) to link your Discord account to your Sentry account."
-ALREADY_LINKED_MESSAGE = "You are already linked to the Sentry account with email: `{email}`."
-UNLINK_USER_MESSAGE = "[Click here]({url}) to unlink your Discord account from your Sentry Account."
-NOT_LINKED_MESSAGE = (
-    "Your Discord account is not linked to a Sentry account. Use `/link` to link your accounts."
-)
-HELP_MESSAGE = """
-`/help`: View this message.
-`/link`: Link your Discord account to your Sentry account to perform actions on Sentry notifications.
-`/unlink`: Unlink your Discord account from your Sentry account.
-"""
+from .types import DiscordResponseTypes
 
```

Comment:
I did a big refactor here. Everything in the one file was giving me bad vibes lol